apps/framework-cli/src/cli/commands.rs

```diff
@@ -58,12 +58,8 @@ pub enum Commands {
     Dev {},
     /// Start a remote environment for use in cloud deployments
     Prod {},
-    /// Generates missing migration files
+    /// Generates sdk for your data models
```

Comment:
This is also used to generate the hash function - so probably not accurate helper

Reply:
Gotcha, will update